Update monitoring docs (#516)
run-ai/docs: docs/admin/runai-setup/maintenance/alert-monitoring.md

       You should see the Prometheus instance(s) listed along with their status.

## Enabling Alertmanager

> In each step, copy the contents of the code snippets to a new file and apply it to the cluster using `kubectl apply -f` command

Reviewer: ```suggestion
In each of the steps in this section, copy the contents of the code snippets to a new file and apply it to the cluster using `kubectl apply -f`.
```

           alertmanagerConfigSelector:
              matchLabels:
                 alertmanagerConfig: runai

2. Validate that your alertmanager instance has started:

        kubectl get alertmanager -n runai

> 3. Validate that prometheus operator has created a service for alertmanager:

Reviewer: ```suggestion
3. Validate that the prometheus operator has created a service for alertmanager:
```

         kubectl get svc alertmanager-operated -n runai

## Configuring Prometheus to Send Alerts

1. Edit the Prometheus configuration:

      `kubectl edit prometheus runai -n runai`

2. Add the following to the `spec.alerting` section:

        alerting:
           alertmanagers:
              - namespace: runai
                name: alertmanager-operated
                port: web

3. Save and exit the editor. The configuration will be automatically reloaded.

## Configuring Alertmanager for Custom Email Alerts

> In each step copy the content to a file and apply it to the cluster using `kubectl apply -f`

Reviewer: ```suggestion
In each step, copy the contents to a file and apply it to the cluster using `kubectl apply -f`
```
What contents? Is there a file with contents? Not sure what you want to say here. This is the start of a section and so we need to be specific about the user needs to copy.

Author: I mean the things in the code snippets, not sure how to refer to them, what do you think?

> In each step, copy the contents to a file and apply it to the cluster using `kubectl apply -f`

Reviewer: If you are referring to code snippiets, then you need to specify that, and more specifically which ones need to be copied, to which file, and to where in the file. If this a new file then that needs to be stated, if it is an already existing file, then you need to be specific about the name of the file.

Author: added

> In each step, copy the contents of the code snippets to a new file and apply it to the cluster using `kubectl apply -f` command

Reviewer: ```suggestion
In each step, copy the contents of the code snippets to a new file and apply it to the cluster using `kubectl apply -f`.
```

1. Add your smtp password as a secret:

        apiVersion: v1
        kind: Secret
        metadata:
           name: alertmanager-smtp-password
           namespace: runai
        stringData:
           password: "your_smtp_password"

> 2. Apply the `alertmanagerconfig`, replace the relevant smtp details with your own (check for indentation problems before applying):

Reviewer: This is backwards:
```suggestion
2.   Replace the relevant smtp details with your own, then apply the `alertmanagerconfig`. Check for indentaion issues before applying.
```

> 2.   Replace the relevant smtp details with your own, then apply the `alertmanagerconfig` using `kubectl apply` command. Check for indentation issues before applying.

Reviewer: ```suggestion
2.   Replace the relevant smtp details with your own, then apply the `alertmanagerconfig` using `kubectl apply`. Check for indentation issues before applying.
```

> 2. Apply the `alertmanagerconfig`, replace the relevant smtp details with your own:

Reviewer: ```suggestion
2. Apply the `alertmanagerconfig`, replace the relevant smtp details with your own:
```
How do you apply? Is there a command for this?

Author: added a description

         apiVersion: monitoring.coreos.com/v1alpha1
         kind: AlertmanagerConfig
         metadata:
           name: runai
           namespace: runai
         labels:
            alertmanagerConfig: runai
         spec:
            route:
               continue: true
               groupBy: 
               - alertname
            
               groupWait: 30s
               groupInterval: 5m
               repeatInterval: 1h

            matchers:
            - matchType: =~
              name: alertname
              value: Runai.*

            receiver: email
         
         receivers:
         - name: 'email'
           emailConfigs:
           - to: '<destination_email_address>'
             from: '<from_email_address>'
             smarthost: 'smtp.gmail.com:587'
             authUsername: '<smtp_server_user_name>'
             authPassword:
               name: alertmanager-smtp-password
               key: password

    !!! Note
        Different receivers can be configured using Alertmanager [receiver-integration-settings](https://prometheus.io/docs/alerting/latest/configuration/#receiver-integration-settings){target=_blank}.

> 3. Save and exit the editor. The configuration will be automatically reloaded.

Reviewer: ```suggestion
3. Save and exit the editor. The configuration will automatically be reloaded.
```
